src/Command/PeekJobCommand.php

```diff
@@ -42,14 +39,7 @@ public function getCommandLine()
      */
     public function parseResponse($responseLine, $responseData)
     {
-        if ($responseLine == ResponseInterface::RESPONSE_NOT_FOUND) {
-            $message = sprintf(
-                '%s: Job %u does not exist.',
-                $responseLine,
-                $this->jobId
-            );
-            throw new Exception\ServerException($message);
-        } elseif (preg_match('#^FOUND (\d+) \d+$#', $responseLine, $matches)) {
+        if (preg_match('#^FOUND (\d+) \d+$#', $responseLine, $matches)) {
```

Comment:
What's the reason for swapping these checks?
Checking for an exact match is faster so it makes sense to do that test first...

Reply:
I changed it when I removed the elseif; order is reverted now.